sample/Pizzeria.Store.Web/Services/IPizzeriaApiClient.cs

```diff
@@ -1,8 +1,10 @@
+using System.Diagnostics.CodeAnalysis;
 using Pizzeria.Store.Contracts;
 using Refit;
 
 namespace Pizzeria.Store.Web.Services;
 
+[SuppressMessage("Performance", "CA1515:Consider making public types internal", Justification = "API client interface should be public for Refit and DI")]

```

Comment:
This suppression shouldn't be needed as there's section in .editorconfig for *.razor.cs and this rule is turned off